types/sass/index.d.ts

```diff
@@ -153,12 +153,18 @@ export interface Options {
     logger?: Logger;
 }
 
+export interface LoggerOptions {
```

Comment:
If we are going to keep this we should name it something more explicit to warnings